app/javascript/api/types/rails/listings/RailsUnit.d.ts

```diff
@@ -21,10 +21,10 @@ type RailsUnit = {
   Unit_Square_Footage?: number
   attributes: ListingAttributes
   isReservedCommunity: boolean
-  Price_Without_Parking?: any
-  Price_With_Parking?: any
-  HOA_Dues_Without_Parking?: any
-  HOA_Dues_With_Parking?: any
+  Price_Without_Parking?: number | string // Eventually these get transformed from numbers into price range strings, to be used in the pricing table
```

Comment:
thought: this seems like a code smell indicating that we're going against the data immutability principle of React.

Reply:
Sigh, I agree... however I prefer this over `any`. With that said, this should be refactored in another ticket to remove the looseness of this type, unless you see an easy way to do it?

Reply:
I don't see an easy way, otherwise I would have offered a suggestion. I can create a ticket to look into the complexities of certain functions like `matchSharedUnitFields`.

Reply:
Okay great, thank you!